doc/system_requirements.md

```diff
@@ -100,14 +100,29 @@ Covers:
 
 Needs: dsn
 
+#### reading-User-Defined Properties
+`req~reading-user-defined-properties~1`
+
+VSCL extracts user-defined Virtual Schema Properties from the Virtual Schema request.
+
+Comment:
+
+Properties are key-value pairs that users can supply with at creation of the Virtual Schema to control the behavior of the VS. While most properties are specific to an individual Virtual Schema, their structure is uniform, so that the general decoding functions are available in VSCL.
```

Comment:
```suggestion
Properties are key-value pairs that users can supply at creation of the Virtual Schema to control the behavior of the Virtual Schema. While most properties are specific to an individual Virtual Schema, their structure is uniform, so that the general decoding functions are available in VSCL.
```